class Foo

	# the following error used to appear on the wrong line number
	
	pro f from var as Foo = Foo()  # .error. arguments
	
	cue init(i as int)  # .warning. related
		pass
		
	def main is shared
		pass
